camera_info_manager/include/camera_info_manager/camera_info_manager.hpp

```diff
@@ -204,6 +205,15 @@ class CameraInfoManager
     rmw_qos_profile_t custom_qos = rmw_qos_profile_default,
     const std::string & ns = "~");
 
+  CAMERA_INFO_MANAGER_PUBLIC
+  CameraInfoManager(
+    rclcpp::node_interfaces::NodeBaseInterface::SharedPtr node_base_interface,
+    rclcpp::node_interfaces::NodeServicesInterface::SharedPtr node_services_interface,
+    rclcpp::node_interfaces::NodeLoggingInterface::SharedPtr node_logger_interface,
+    const std::string & cname = "camera", const std::string & url = "",
+    rclcpp::QoS custom_qos = rclcpp::SystemDefaultsQoS(),
```

Comment:
Avoid signature conflict here
```suggestion
    rclcpp::QoS custom_qos,
```